new_release.sh

```diff
@@ -0,0 +1,102 @@
+#!/bin/bash
+
```

Comment:
move to a `scripts` or `tools` directory instead of littering the root of the repository.